ui/src/app/utils/stores.tsx

```diff
@@ -27,6 +33,19 @@ export const authStore = atom<AuthStore>({authLoaded: false, isSignedIn: false})
 
 export const cdrVersionStore = atom<CdrVersionTiersResponse>({tiers: []});
 
+export interface GenomicExtractionStore {
+  [workspaceNamespace: string]: GenomicExtractionJob[];
```

Comment:
I don't think this warrants changing since we apply this pattern elsewhere but I'm wondering if we're needlessly giving our stores the ability to handle multiple workspaces by defining it as a map. It adds a little boilerplate to every usage of the store and we don't have any components that actually need the context of more than a single workspace. They just need to get the extraction jobs that are currently relevant.

The alternative solution I see is to have our stores return the value object without needing a workspaceNamespace key and the stores can subscribe to a change in the current workspace context and update its value if it detects a change.

I'd have to actually try it out to see if it feels nicer but curious to hear your thoughts.

Reply:
I like the idea; I'll spend ~half an hour futzing with it as part of this PR to see if I get traction; if not I'll file a follow up spike ticket